plugins/modules/packaging/os/apk.py

```diff
@@ -61,6 +61,12 @@
       - Upgrade all installed packages to their latest version.
     type: bool
     default: no
+  world:
+    description:
+      - Use a custom world file when checking for explicitly installed packages
```

Comment:
```suggestion
      - Use a custom world file when checking for explicitly installed packages.
```

Reply:
I thought I did that, thanks for checking :+1: